src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -257,6 +258,37 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchFeedItems();
     };
 
+    updateComment = (
+        id: string,
+        text: string,
+        hasMention: boolean,
+        permissions: BoxItemPermission,
+        onSuccess: ?Function,
+        onError: ?Function,
+    ): void => {
+        const { file, api, onCommentUpdate = noop } = this.props;
+
+        const errorCallback = (e, code) => {
```

Comment:
Should these be separate methods to stay consistent with the other top-level actions?

Reply:
I don't know what you mean, most of the API methods in the file define the callbacks this way